fsspec/implementations/tests/test_jupyter.py

```diff
@@ -38,6 +38,7 @@ def jupyter(tmpdir):
         P.terminate()
 
 
+@pytest.mark.skipif(WIN, reason="Subprocess gets stuck in a loop")
```

Comment:
I didn't notice this before. Any idea what's going on?

Reply:
I found the problem - Something to do with the temp dir passed to Jupyterlab in the launch call showing up as an invalid path. Surrounding the path with "" fixed it. The test now passes when I run it on Windows.

```
cmd = f'jupyter notebook --notebook-dir="{tmpdir}" --no-browser --port=5566'
```

Reply:
This line was to be removed according to the thread above

Reply:
Quite right - I made the changes but forgot to push them. It is done now.